src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -181,6 +182,22 @@ trait BackendCall {
       Future.successful(ExecutionHash("", None))
   }
 
+  def pollingInitialInterval: FiniteDuration
+  def pollingMaxInterval: FiniteDuration
+  def pollingMultiplier: Double
+  def pollingInitialGap: FiniteDuration
```

Comment:
Also I think we should be more careful in general about default values in traits, it's often done as a convenience here but it can lead to confusion as you don't need to override it, so you might not realize you left something undefined.